### PR TITLE
Create accordions logic for Covid Page

### LINKED ISSUE #103: Create accordions logic for Covid Page
Current site has a COVID-19 page for resources: https://www.oregonsadventurecoast.com/covid-19/

Client wishes to be able to add a set of accordions inspired from the Kissimee COVID page here: https://www.experiencekissimmee.com/covid-19-cancellations-closures

1. Create Front Matter Template which uses most of the default Front Matter + The accordion fields.
2. Create Accordion Front Matter Repeatable Group Field:
``` 
    Title
    Repeatable Group Field
        Title
        WYSIWYG
    Title
    Repeatable Group Field
        Title
        WYSIWYG
ETC...
```
This way, editors can add as many groups of accordion like the kissimme page.

We should try a minimize the JS footprint for this. Adding a simple `is-expanded` class on the item when header is clicked would be plenty. CSS will take care of the rest.

As for where any given accordion should be displayed on the page. As client might need this on every pages I think we should use a shortcode for now. 

Shortcode wil grab the accordion data of the page and print the accordion wherever editor see fit.

### PR DESCRIPTION
**Pull request by mountainbug95**
Fixes #103

### CODE REVIEW

File: assets/sass/modules/_accordion-sections.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                    right: -40px;
+                    top: 3px;
+                }
+            }
+
+            span{
+                position: relative;
+                color: $white;
+                font-size: 20px;
+                font-weight: 600;
```
Comment (regisphilibert) [inline assets/sass/modules/_accordion-sections.scss]: I think the font weight should be regular here.

**Comment by regisphilibert** [comment]
@mountainbug95 one more thing. The chevron seem to disappear when we expand, is it easy to have them rotate?

**Comment by mountainbug95** [comment]
@regisphilibert I'm not sure I understand the issue with the chevron. I've added a transition to it so it will animate (as opposed to immediately shift from pointing up to pointing down.) Is that what you meant?

**Comment by regisphilibert** [comment]
@mountainbug95 No the chevron disappeared when I clicked on the box (might have been a one time/one browser) kind of thing. But the animation works great thanks!

**Pull request merged**